godtools/App/Features/ToolDetails/Subviews/ToolDetailsVersionsView.swift

```diff
@@ -16,6 +16,12 @@ struct ToolDetailsVersionsView: View {
         
         VStack(alignment: .leading, spacing: 0) {
             
+            Text(viewModel.versionsMessage)
+                .foregroundColor(ColorPalette.gtGrey.color)
+                .font(FontLibrary.sfProTextRegular.font(size: 16))
+                .multilineTextAlignment(.center)
```

Comment:
I think the text needs to be aligned left?

Reply:
Yes, that is supposed to be left aligned.